src/ui/actions/session.ts

```diff
@@ -62,6 +61,8 @@ export function getAccessibleRecording(
         if (expectedError) {
           dispatch(setExpectedError(expectedError));
           return null;
+        } else {
+          dispatch(selectors.clearExpectedError());
```

Comment:
This seems suspect to me since it could be clearing other errors. Alternatives welcome